tenders/hvt/hvt_openbsd.h

```diff
@@ -32,6 +32,69 @@ struct hvt_b {
     int      vmd_fd;
     uint32_t vcp_id;
     uint32_t vcpu_id;
+    struct   vcpu_reg_state *vrs;
 };
 
+/*
+the following code has been extracted from musl
```

Comment:
Is there really no definition of `prstatus_t` available to userland on OpenBSD? Where does OpenBSD GDB (ports or otherwise get it from)? It's understandable that an OpenBSD GDB will not like a "Linux-like" core file, and, in any case, I'd like to avoid carrying a definition of `prstatus_t` and friends around.

Reply:
From what I can find and from chats on bsd.network, no. I will investigate further, if and where OpenBSD GDB gets it from.  Yes, I agree, i will see what i can determine.